### ISSUE TITLE
Fillers should be hidden to people using screenreaders

### ISSUE THREAD

**Issue by codynguyen1116** [DrewDahlman/Mason#60]
The fillers simply have no content and are for simple aesthetic purposes: filling in empty spaces and show as white/transparent rectangles. For this reason, they should automatically have `aria-hidden="true"` added to these elements so the MasonJS grid system is more accessible. :)

**Comment by DrewDahlman** [comment]
This is the case with the examples but the intended use of fillers would have content in them. Many people use the filler objects as spare objects and they are placed into the grid as actual content pieces. Appreciate the input though - maybe there is something that could be done to detect if there isn't any content, but I worry that may be a bit too heavy handed.

**Comment by julien-gargot** [comment]
Out of subject, but maybe useful to know and a little bit related to screen-readers’ question:
if you do not want any *fillers* in your HTML to simply play around with the grid, use an empty `filler` option :
```
filler: {}
``` 
;)
